Assets/Scripts/ScriptableObjects/PlayerInformation.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using UnityEngine;
+
+namespace ScriptableObjects
+{
```

Comment:
This technically doesn't need it's own namespace, and it also isn't a ScriptableObject. That its own completely separate bag of worms! Removal of the namespace declaration requested.